package.json

```diff
@@ -147,7 +147,8 @@
     "build:test": "npm-run-all test linters",
     "build:release": "npm-run-all build build:test",
     "build:ci": "npm-run-all sweep build:release",
-    "build:api": "wca analyze 'src/auro-flight*.js' --outFiles docs/api.md",
+    "build:api": "wca 'scripts/wca/*' 'src/auro-flight-*.js' --outFiles docs/api.md",
+    "build:api:prepare": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/generateWcaComponent.mjs 'src/auro-flight.js'",
```

Comment:
Need to remove line 151. We can reference how to run the script from the Auro Library readme.